admin_honeypot/urls.py

```diff
@@ -1,9 +1,9 @@
 from admin_honeypot import views
-from django.urls import path, re_path
+from django.urls import re_path
 
 app_name = 'admin_honeypot'
 
 urlpatterns = [
-    path('login/', views.AdminHoneypot.as_view(), name='login'),
+    re_path(r'^login/$', views.AdminHoneypot.as_view(), name='login'),
```

Comment:
Using the original `path` is likely going to be faster. Avoid `re_path` where possible.